src/wp-includes/class-wp-taxonomy.php

```diff
@@ -452,4 +452,41 @@ public function add_hooks() {
 	public function remove_hooks() {
 		remove_filter( 'wp_ajax_add-' . $this->name, '_wp_ajax_add_hierarchical_term' );
 	}
+
+
+	/**
+	 * Gets the REST API controller for this taxonomy.
+	 *
+	 * Will only instantiate the controller class once per request.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return WP_REST_Controller|null The controller instance, or null if the taxonomy
+	 *                                 is set not to show in rest.
+	 */
+	public function get_rest_controller() {
+		if ( ! $this->show_in_rest ) {
+			return null;
+		}
+
+		$class = $this->rest_controller_class ? $this->rest_controller_class : WP_REST_Terms_Controller::class;
+
+		if ( ! class_exists( $class ) ) {
+			return null;
+		}
+
+		if ( ! is_subclass_of( $class, WP_REST_Controller::class ) ) {
+			return null;
+		}
+
+		if ( ! $this->rest_controller ) {
```

Comment:
We need to introduce `$rest_controller` as a property of `WP_Taxonomy`.

Reply:
It's my fault...when I looked to see if it had the same property, I mixed up rest_controller and rest_controller_class

Reply:
Should we introduce it? It makes sense that WP_Taxonomy should mirror WP_Post_Type in this regard, but is it within scope?

Reply:
I don't have an issue with introducing this for parity in this ticket.